src/state/user/reducer.ts

```diff
@@ -164,4 +167,8 @@ export default createReducer(initialState, builder =>
     .addCase(toggleURLWarning, state => {
       state.URLWarningVisible = !state.URLWarningVisible
     })
+    .addCase(updateUserAdvancedSwapDetails, (state, action) => {
+      state.userAdvancedSwapDetails = action.payload.userAdvancedSwapDetails
+      state.timestamp = currentTimestamp()
```

Comment:
To be honest I don't know. There is a `timestamp` property that is updated every time we update a value. I think it could be there for client side debugging, but I'm not sure. Should we delete it?

Reply:
Fixed!